.gitattributes

```diff
@@ -2,4 +2,6 @@
 *.cs diff
 *.csproj diff
 *.sln diff
-*.resx diff
\ No newline at end of file
+*.resx diff
+
+* text=auto
```

Comment:
Is this intended? Please open a separate PR for this if you wanted this changed

Reply:
It is- the line ending difference is what was causing me issues with the .resx files (all the strings related files, really). Every time they get regenerated, it's as CRLFs since I'm on Windows, and git was trying to push those as changes. text=auto tells git to check in files with the same line endings as they were checked out with.